src/components/ServiceList.jsx

```diff
@@ -0,0 +1,27 @@
+import FilterList from "../components/FilterList";
+import React from "react";
+import ServiceCard from "../components/ServiceCard";
+import useServices from "../hooks/useServices";
+
+const ServiceList = () => {
+  const apiResponse = useServices();
```

Comment:
not going to blow but this can be simplified to.

```js
const { services = [], isLoading } = useServices();
```

Reply:
done